imputeme/v1.0.4/Dockerfile

```diff
@@ -0,0 +1,225 @@
+LABEL about.license_file="https://github.com/lassefolkersen/impute-me/blob/master/LICENSE"
+LABEL about.license="GNU Lesser General Public License v3.0"
+LABEL extra.identifiers.biotools="impute-me"
+LABEL about.tags="Genomics, polygenic risk scores, personal genomics"
+
+################## MAINTAINER ######################
+
+MAINTAINER Lasse Folkersen, lassefolkersen@impute.me
+
+
+################## INSTALLATION ######################
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install easy R-packages
+RUN R -e "install.packages(c( \
+'DT', \
+'igraph',  \
+'jsonlite', \
+'mailR', \
+'openxlsx', \
+'R.utils', \
+'visNetwork', \
+'docstring' \
+),dependencies=TRUE, repos = 'http://cran.rstudio.com/')"
+
+#Install basic non-problematic apt-get apps
+RUN apt-get update && apt-get -y install \
+wget \
+git \
+sed \
+tar \
+unzip \
+vim \
+curl \
+vcftools \
+gawk
+
+#Install kandinsky package, plus the remotes-package that is needed to get it
+RUN R -e "install.packages(c( \
+'remotes' \
+),dependencies=TRUE, repos = 'http://cran.rstudio.com/')"
+RUN R -e "remotes::install_github('gsimchoni/kandinsky')"
+
+#Install the mailR, plus the java environment required for it
+RUN apt-get -y update && apt-get install -y \
+default-jdk \
+r-cran-rjava 
+RUN R -e "install.packages(c( \
+'rJava', \
+'mailR' \
+),dependencies=TRUE, repos = 'http://cran.rstudio.com/')"
+
+#Install plotly - takes forever and often fails in non-deterministic ways. 
+#All it affects is the 3D-plotting of ancestry. Doesn't affect calculations at all.
+# RUN R -e "install.packages(c( \
+# 'plotly' \
+# ),dependencies=TRUE, repos = 'http://cran.rstudio.com/')"
+
+#configure the shiny_server.conf
+RUN sed -i 's=run_as shiny=run_as ubuntu=' /etc/shiny-server/shiny-server.conf && \
+sed -i 's=site_dir /srv/shiny-server=site_dir /home/ubuntu/srv/impute-me/=' /etc/shiny-server/shiny-server.conf && \
+sed -i 's=log_dir /var/log/shiny-server=log_dir /home/ubuntu/logs/shiny=' /etc/shiny-server/shiny-server.conf && \
+sed -i 's=directory_index on=directory_index off=' /etc/shiny-server/shiny-server.conf
+
+#Install cron stuff using the supercronic app (needs root to install, then can run as user)
+ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.11/supercronic-linux-amd64 \
+SUPERCRONIC=supercronic-linux-amd64 \
+SUPERCRONIC_SHA1SUM=a2e2d47078a8dafc5949491e5ea7267cc721d67c
+RUN curl -fsSLO "$SUPERCRONIC_URL" \
+&& echo "${SUPERCRONIC_SHA1SUM}  ${SUPERCRONIC}" | sha1sum -c - \
+&& chmod +x "$SUPERCRONIC" \
+&& mv "$SUPERCRONIC" "/usr/local/bin/${SUPERCRONIC}" \
+&& ln -s "/usr/local/bin/${SUPERCRONIC}" /usr/local/bin/supercronic
+
+
+#The impute.me server runs as ubuntu default user, so the docker should too
+RUN useradd ubuntu
+RUN mkdir /home/ubuntu
+RUN chown ubuntu /home/ubuntu && \
+chown ubuntu /var/lib/shiny-server
+USER ubuntu
+WORKDIR /home/ubuntu
+
+#Create the directory stucture
+RUN mkdir /home/ubuntu/srv && \
```

Comment:
should avoid installing things in /home, with singularity for example, home is mounted over container home